Phonebook/phonebook/templates/frontend-ingress.yaml

```diff
@@ -15,8 +15,7 @@ metadata:
   {{- end }}
 spec:
   rules:
-    - host: {{ .Values.host | quote }}
-      http:
+    - http:
```

Comment:
why removing the host

Reply:
In order to make the app available on every possible host. The are automatically redirected to the host given in the values.yml